packages/core/src/stylable-value-parsers.ts

```diff
@@ -58,8 +45,8 @@ export const valueMapping = {
     root: '-st-root' as const,
     states: '-st-states' as const,
     extends: '-st-extends' as const,
-    mixin: '-st-mixin' as const,
-    partialMixin: '-st-partial-mixin' as const,
+    mixin: '-st-mixin' as const, // ToDo: change to STMixin.MixinType.ALL,
```

Comment:
I think we can deprecate these or remove. 
remove only usages in [SE](https://github.com/wixplosives/stylable-editor/pull/793/files)